zowe/zowe-install-packaging
# Add ZIS plugin install automation as part of zwe component install

Signed-off-by: Leanid Astrakou <lastrakou@rocketsoftware.com>



Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR. If you don't know all of this information when you create this PR, don't worry. You can edit this template as you're working on it.

- [ ] Necessary documentation (if appropriate) have been added / updated
- [x] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 

#### Relevant issues


Fixes 

#### Changes proposed in this PR

- 

#### Does this PR introduce a breaking change?



How to Test?

1. Get ZIS plugin (one with /loadlib and /samplib)
2. Construct your component to include ZIS plugin (see documentation for ZIS plugin schema here: https://github.com/zowe/docs-site/blob/6f533c7ac3e3d0cb3ff83a5a8b29626abe051a86/docs/appendix/server-component-manifest.md - see `zisPlugins`)
3. Run zwe components install on component with ZIS plugins
